hypervisor/src/mshv/mod.rs

```diff
@@ -53,6 +53,12 @@ pub use {
 
 pub const PAGE_SHIFT: usize = 12;
 
+// These constants are derived from GHCB spec page 26.
+// Link: https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/specifications/56421.pdf
+pub const GHCB_RAX_OFFSET: u64 = 0x01F8;
```

Comment:
I would like to have these live in a separate module just for GHCB definitions if in the future there are more constants coming. That's not a blocker right now.

Reply:
Yeah, for the time being we don't have any more constants coming in. So, I will keep this as a future task when we starting porting this work on KVM in future.